ci-cd/github-actions.md

```diff
@@ -148,13 +148,16 @@ step:
 
 {% raw %}
 ```yaml
-      -
-        name: Login to Docker Hub
-        uses: docker/login-action@v2
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v1
         with:
           username: ${{ secrets.DOCKER_HUB_USERNAME }}
           password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
-      -
+      
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      
```

Comment:
* Why downgrading to `docker/login-action@v1`?
* `-` missing for `Build and push`
* Please keep the layout as it is and just add the `Set up Docker Buildx` step in your changes.